scripts/caravan/event-handlers/action.lua

```diff
@@ -143,7 +179,16 @@ local function on_confirmed(event)
         action.item_count = value
     end
 
-    if player.gui.screen[prefix .. "_frame"] then 
+    -- keep slider in sync after setting the action value 
+    local frame = player.gui.screen[prefix .. "_frame"]
+    if frame and frame.valid then
```

Comment:
Sorry I missed this the first time, I don't think the .valid is necessary here since the element will be removed from the table if it isn't valid.

Reply:
Done